api/Navigator.json

```diff
@@ -309,9 +323,6 @@
             "opera_android": {
               "version_added": "27"
             },
-            "qq_android": {
-              "version_added": null
-            },
```

Comment:
I generally prefer to keep these

Reply:
Then should we update it everywhere? I copied from the SW and these did not appear anywhere.

Reply:
Well, the default value for keys that aren’t present is `null`, so adding `qq_android`, `uc_android` and `uc_chinese_android` to the rest isn’t necessary, but removing them from here can potentially lead to merge conflicts, should a PR which adds data for them get merged before this.